README.md

```diff
@@ -119,6 +119,12 @@ nameOverride:
 # Override the full name of the deployment. When empty, the name will be "{release-name}-{chart-name}" or the value of nameOverride if specified
 fullnameOverride:
 
+# Additional labels to be added to all deployed resources by the chart
+additionalLabels: {}
+  # tags.datadoghq.com/env: "<ENV>"
```

Comment:
Since datadog isn't a service that is created from the helm-chart, it may be preferable to make the label more generic. 

Non-blocking

Reply:
make sense